src/appmixer/hibob/service.json

```
@@ -0,0 +1,10 @@
+{
+    "name": "maesn.hibob",
```

Appmixer vendor is expected when the connector is in `src/appmixer` folder.
```suggestion
    "name": "appmixer.hibob",
```